### ISSUE TITLE
Zooming and scaling does not work properly on mobile devices

### ISSUE THREAD

**Issue by jakubmiarka** [alphagov/tech-docs-gem#110]
The scaling and zooming on mobile devices is disabled:
https://github.com/alphagov/tech-docs-gem/blob/6c44ea2c6e8787046af0fe4fa6049c9609f2beab/lib/source/layouts/core.erb#L6

This fails [WCAG 2.1 - 1.4.4 Resize text Level AA criterion](https://www.w3.org/WAI/WCAG21/quickref/#resize-text).

**Comment by 36degrees** [comment]
From memory, this is because of the sticky 'table of contents' on mobile, and the peculiar way that sticky elements scale and position themselves when zooming (I think there may also have been issues with the table of contents overlay that appears)

As a lot of the tech docs using this format are now split across multiple pages, it may be that the table of contents overlay can be redesigned or removed, but this isn't a trivial change and IMHO needs to be designed and researched.

**Comment by nickcolley** [comment]
Testing this on Android Chrome, it lets you zoom regardless of if we've turned it off.

Will need to test on more devices to see what the behaviour is.
